common/arg.cpp

```diff
@@ -34,6 +34,14 @@ common_arg & common_arg::set_env(const char * env) {
     return *this;
 }
 
+static void set_ms_env() {
+#ifdef _WIN32
+    putenv("LLAMACPP_USE_MODELSCOPE=True");
+#else
+    setenv("LLAMACPP_USE_MODELSCOPE", "True", 1);
+#endif
+}
+
```

Comment:
IMO this is not a good pattern, user application should not set an arbitrary env var unless it's absolute needed (for example, modifying system-level params)